deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1642,52 +1656,32 @@ def fetch_and_check(self, event_type, min_timeout=1, triggers_event=True, extra_
             elif event_type == 'added':
                 self.events = self.fetch_events(min_timeout, triggers_event, extra_timeout, error_message=error_msg)
                 self.check_events(event_type)
-
-                # The callback for the registry parent key will be `None` when `check_mtime` is disabled
-                if self.registry_dict[self.registry_key][1] is not None:
-                    self.events = self.fetch_events(min_timeout, triggers_event, extra_timeout,
-                                                    error_message=key_error_msg, is_parent=True)
-                    self.check_events(event_type, check_parent_key=True)
             elif event_type == 'deleted':
-                if self.registry_dict[self.registry_key][1] is not None:
-                    self.events = self.fetch_events(min_timeout, triggers_event, extra_timeout,
-                                                    error_message=key_error_msg, is_parent=True)
-                    self.check_events(event_type, check_parent_key=True)
-
                 self.events = self.fetch_events(min_timeout, triggers_event, extra_timeout, error_message=error_msg)
                 self.check_events(event_type)
 
-        def fetch_events(self, min_timeout=1, triggers_event=True, extra_timeout=0, error_message='', is_parent=False):
+        def fetch_events(self, min_timeout=1, triggers_event=True, extra_timeout=0, error_message=''):
             try:
-                if is_parent:
-                    result = self.log_monitor.start(timeout=min_timeout,
-                                                    callback=self.registry_dict[self.registry_key][1],
-                                                    timeout_extra=extra_timeout,
-                                                    encoding=self.encoding,
-                                                    error_message=error_message).result()
-                else:
-                    # We substract 1 to the length of the dictionary because the parent key should not be detected here
-                    result = self.log_monitor.start(timeout=max((len(self.registry_dict) - 1) * 0.01, min_timeout),
-                                                    callback=self.callback,
-                                                    accum_results=len(self.registry_dict) - 1,
-                                                    timeout_extra=extra_timeout,
-                                                    encoding=self.encoding,
-                                                    error_message=error_message).result()
-
-                assert triggers_event, f'No events should be detected.'
+                result = self.log_monitor.start(timeout=max((len(self.registry_dict)) * 0.01, min_timeout),
```

Comment:
It should be recommended to replace hardcoded `0.01` value with a variable with a meaningful name 
```suggestion
                result = self.log_monitor.start(timeout=max((len(self.registry_dict)) * timeout_per_registry_estimation, min_timeout),
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1670/commits/c5f646dba669a1facacd74025ba515b4b452bda6)